### DISCUSSION TITLE
开启 dynamicImport 后，chunk里的publicpath不正确

### DISCUSSION THREAD

**Discussion by NgeKaworu** [umijs/umi#7558] [Q&A]
base: '/micro/user-center',
 publicPath: '/micro/user-center/',
 
http://localhost:8000/micro/vendors~layouts__

chunk只拼了前端一截

**Comment by dapandocs** [comment]
我也是这个问题。。。

**Comment by dapandocs** [comment]
楼主，解决了吗？有解决办法吗
